Docs: Adds pgnet and database webhooks (#9803)
supabase/supabase: apps/reference/docs/guides/database/extensions/pgnet.mdx

---
id: pgnet
title: 'pg_net: Async Networking'
description: 'pg_net: an async networking extension for PostgreSQL.'
---

> 

Reviewer: Could we add a notice here that pg_net is still not stable yet? The interface will likely change to the one proposed on https://github.com/supabase/pg_net/issues/62 and functions will not require security definer.

```suggestion
:::caution

pg_net is not at 1.0(stable) yet. Expect the function signatures to change.
```

---
id: pgnet
title: 'pg_net: Async Networking'
description: 'pg_net: an async networking extension for PostgreSQL.'
---

import Tabs from '@theme/Tabs'
import TabItem from '@theme/TabItem'

:::caution

> The pg_net API is not at 1.0 (stable). Functions signatures may change.

Reviewer 2: Maybe this should say it's still alpha/beta instead so it sounds a bit less negative.

> The pg_net API is not at 1.0 (stable). Functions signatures may change.

Author: ```suggestion
The pg_net API is in beta. Functions signatures may change.
```

---
id: pgnet
title: 'pg_net: Async Networking'
description: 'pg_net: an async networking extension for PostgreSQL.'
---

import Tabs from '@theme/Tabs'
import TabItem from '@theme/TabItem'

:::caution

The pg_net API is not at 1.0 (stable). Functions signatures may change.

:::

> 

Author: ```suggestion
```
